docker/docker-compose.yml

```diff
@@ -64,6 +64,7 @@ services:
     working_dir: /app
     volumes: 
       - "../web:/app"
+      - /app/node_modules
```

Comment:
This doesn't solve the issue, nor is it the behavior we want

Reply:
Also, this would give it an empty node modules directory iirc

Reply:
Yes, that's exactly the behavior we want so it doesn't get mirrored over to the host.

Reply:
1. that's not the behavior we want, it should get mirrored. this is a dev environment
2. we don't want people to have to exec into it to install packages
3. this still doesnt solve the original issue lol

Reply:
1. Except that the mirroring leaves files owned by root. Meaning it is the behavior you want. 
2. they won’t have to - you have to restart the docker container when you install packages anyways, and the restart will install the missing packages
3. yes it does, because node modules was the only directory owned by root in the web directory on your install on scholes.

Reply:
1-2: we didnt have to before. i'm okay with doing it now if it fixes the issue, but see 3
3: no, that's not true. you didn't check the worktree "sandbox"

Reply:
1. sorry?
2. i don't konw how but ok
3. i did. did you notice the other lines i changed?

Reply:
for 3, can u point it out? i might've missed but i dont see anything directly pertaining to that

Reply:
![image](https://github.com/northwesternfintech/NUTC/assets/76668284/04c44edd-52dc-4e3a-a7d5-8c94ce6aed4b)

try running it and tell me if any files are left behind again

Reply:
oh i forgot to mention grafana_data, thats what i was referring to. thats the on we actually need a solution for soon

Reply:
can you tackle that or should i @egelja